packages/core/src/utils/defaults.ts

```diff
@@ -0,0 +1,66 @@
+import { type Address, type Chain } from "viem";
+import {
+  arbitrum,
+  arbitrumGoerli,
+  base,
+  baseGoerli,
+  goerli,
+  mainnet,
+  optimism,
+  optimismGoerli,
+  polygon,
+  polygonMumbai,
+  sepolia,
+} from "viem/chains";
+
+/**
+ * Utility method returning the entry point contrafct address given a {@link Chain} object
+ *
+ * @param chain - a {@link Chain} object
+ * @returns a {@link abi.Address} for the given chain
+ * @throws if the chain doesn't have an address currently deployed
+ */
+export const getDefaultEntryPointContract = (chain: Chain): Address => {
```

Comment:
nit(naming): to me the method name implies retrieving an object containing details about a contract, not just the address.

should we name it: `getDefaultEntryPointAddress`